### ISSUE TITLE
Inactivity

### ISSUE THREAD

**Issue by upintheairsheep** [shonumi/shonumi.github.io#3]
Why were you inactive? Is something big happening... Or is it just a small break?

**Comment by shonumi** [comment]
I'm working on something big at the moment, and it's taking a lot of time and effort (alongside trying to improve GBE+ and it's documentation on a daily basis). It's a pretty large undertaking, so I don't have a lot of energy to spare on the Edge of Emulation articles (even though I have 4 topics queued up).

I'll announce it when it's done (or almost done). It's getting down to just a few more months, but I do plan on releasing a new article soon in the meantime.

**Comment by upintheairsheep** [comment]
Thank you. Did you see my email about the smart tv, I think Samsung smart
tv 2012 deserves to be emulated.

**Comment by shonumi** [comment]
Yes, I saw your suggestions, but unfortunately I haven't had time to respond to any emails promptly these past few months. Thanks for pointing out some potential projects! For the foreseeable future, however, I plan to stick to the few remaining pieces of Game Boy hardware left to cover and then move onto the DS (or other handhelds gaming consoles). I'll let people know if I start expanding elsewhere.

**Comment by upintheairsheep** [comment]
Make sure to try to aquire the Workboy or at least a technical documentation

**Comment by upintheairsheep** [comment]
Btw if you plan on emulating TVs you can start off with an Angry Birds special edition Roku 2 with the game pre installed
https://www.ebay.com/p/135053052

**Comment by upintheairsheep** [comment]
For MY childhood… it was an iPad 2. This would be a wild ride to preserve millions of childhoods because I realized I wanted to show my grandkids how my childhood was like. To prevent copyright, download the IPSW from Apple so it is legal. The fun part is gonna be accessory emulation. Unlike your other consoles, this one is FULLY unemulated and if not emulated, will be lost to time or become rare as f-k. You might wanna emulate the OG iPhone first, or after because it will get a lot easier. This is a list of all accessories and devices by them: https://apple.fandom.com/wiki/List_of_Apple_products 

Here’s a checklist…

- [ ]  Get core ready 

- [ ] CPU Emulation 
- [ ] GPU Emulation 
- [ ] Mainboard fully emulated
- [ ] Screen output emulation
- [ ] Touch input emulation 
- [ ] Speaker - L
- [ ] Speaker - R
- [ ] Microphone
- [ ] Camera - Front (selfie)
- [ ] Camera - Back (landscape)
- [ ] Home button
- [ ] Volume up button
- [ ] Volume down button
- [ ] Power button
- [ ] Successfully boot recovery mode (the iTunes screen of death I like to call it)
- [ ] Successfully Boot iOS!!! (Your gonna need a party or some fast food to celebrate this)
- [ ] NAND Reading
- [ ] NAND writing
- [ ] Make a USB-C to 30-pin adapter to research and use original hardware
- [ ] Headphone Jack emulation 
- [ ] Headphone Jack forwarding (to pc headphone Jack like remote controls)
- [ ] WiFi Emulation 
- [ ] Bluetooth emulation 
- [ ] WiFi forwarding
- [ ] Bluetooth forwarding 
- [ ] 30-pin emulation 
- [ ] 30-pin forwarding 
- [ ] Cellular emulation
- [ ] Insert a fake SIM card with a satire carrier name (like Amogus Wireless)
- [ ] Battery percentage emulation
- [ ] Connect the iPad 2 emulator to a Mac or a Mac Virtual Machine and it shows up in iTunes or Finder
- [ ] Connect the iPad 2 emulator to your Linux PC
- [ ] Connect the emulator to a VM of Windows 
- [ ] Successfully jailbreak the Emulator 
- [ ] Successfully install a .ipa file to the Emulator
- [ ] Use a compressed virtual drive

Now the accessories (the fun part):

- [ ]
